tests/kitchen/tests/wordpress/_modules/wordpress.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+'''
+Manage wordpress plugins
+'''
```

Comment:
If this is going to be used as the example for how to write tests, I think we may wish to add substantially more in-line documentation to this module

Reply:
comments added to the tests

I moved the module and state to the actual salt modules and states, since they might be useful to someone, and added cli examples and a state example to each function.